Added API for observing properties easily. (#12)
timofurrer/observable: README.md

[]
```

## Usage of ``observable.property.ObservableProperty``

A property that can be observed easily by listening for some special,

> auto-generated events. It's API is identical to that of the build-in

Reviewer: ```suggestion
auto-generated events. Its API is identical to that of the build-in
```

``property``.

The ``ObservableProperty`` needs an ``Observable`` object for
triggering events. If the property is an attribute of an object of
type ``Observable``, that one will be used automatically. To specify a
different ``Observable`` object, pass it as the ``observable`` keyword
argument when initializing the ``ObservableProperty``. You may also pass

> a string for ``observable``, it's then looked for an attribute of that

Reviewer: ```suggestion
a string for ``observable``, which looks for an attribute of that
```
